lib/mapping.js

```diff
@@ -16,16 +16,20 @@ module.exports = {
   ircToSlack: function(originalMessage) {
     var modifiedMessage = originalMessage;
     _.each(userMap, function(val, key, list) {
-      var re = new RegExp("@?" + key, 'gi');
+      var re = new RegExp("@?" + key + '[^a-zA-Z]', 'i');
```

Comment:
Do we want to do the same thing before the word?  to prevent something like "throb" from triggering?  Also, if we do that, do we really need the urlRegex?